website/docs/r/policy_context_profile.html.markdown

```diff
@@ -0,0 +1,77 @@
+---
+layout: "nsxt"
+page_title: "NSXT: nsxt_policy_context_profile"
+sidebar_current: "docs-nsxt-resource-policy-context-profile"
+description: A resource to configure a Context Profile.
+---
+
+# nsxt_policy_context_profile
+
+This resource provides a method for the management of a Context Profile.
+ 
+## Example Usage
+
+```hcl
+resource "nsxt_policy_context_profile" "test" {
+    display_name      = "test"
+    description       = "Terraform provisioned ContextProfile"
+    attribute {
+        data_type   = "STRING"
+        description = "test-domain-name-attribute"
+        key         = "DOMAIN_NAME"
+        value       = ["*-myfiles.sharepoint.com"]
+    }
+    attribute {
+        data_type   = "STRING"
+        description = "test-app-id-attribute"
+        key         = "APP_ID"
+        value       = ["SSL"]
+        sub_attribute {
+            data_type = "STRING"
+            key       = "TLS_VERSION"
+            value     = ["SSL_V3"]
+        }
+    }
+}
+
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `display_name` - (Required) Display name of the resource.
+* `description` - (Optional) Description of the resource.
+* `tag` - (Optional) A list of scope + tag pairs to associate with this resource.
+* `nsx_id` - (Optional) The NSX ID of this resource. If set, this ID will be used to create the resource.
+* `attribute` - (Required) A repeatable block to specify attributes for the context profile. At least one block is required.
+  * `datatype` - (Required) Datatype for `attribute`, must be `STRING`.
```

Comment:
is there a reason to expose it then?

Reply:
I was thinking other datatypes may be allowed in the future, so I exposed this. On second thought this is indeed somewhat redundant as of now. Let me remove this config then. Thanks!